### DISCUSSION TITLE
Doc: Ability to programmatically change editor value

### DISCUSSION THREAD

**Discussion by ukch** [outline/rich-markdown-editor#231] [General] [answered]
Follow-up from #120 

I see that with the migration to prosemirror, the above issue is marked as closed. However, I can't see any way to do this in the documentation, and it's not clear to me how to make the best use of prosemirror's internals and this tool to do what I want. Would it be possible to add some examples of how to utilise prosemirror to do some more advanced configuration of this tool?

**Comment by tommoor** [comment, marked as answer]
Howdy, just change the `value` prop like you would any other input. It's covered in the example code, here specifically:

https://github.com/outline/rich-markdown-editor/blob/develop/example/src/index.js#L68-L74

**Reply by ukch** [reply]
Amazing! That's so simple I didn't even consider it as an option! :man_facepalming:

**Reply by tommoor** [reply]
I added a section in the README:
https://github.com/outline/rich-markdown-editor/blob/develop/README.md#value
